src/component/menu/__tests__/routes.test.tsx

```diff
@@ -6,5 +6,5 @@ test('returns all baseRoutes', () => {
 
 test('getRoute() returns named route', () => {
     const featuresRoute = getRoute('/features');
-    expect(featuresRoute.path).toEqual('/features');
+    expect(featuresRoute?.path).toEqual('/features');
```

Comment:
Should this ever not exist?

Reply:
It shouldn't but that's the type, unless you use `routes = {...} as const` and do some fancy inference on `getRoute`, ensuring that first parameter is not an arbitrary string.